_mobile/home.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: mobile
+title: Home
+permalink: /mobile/home/
+background-class: mobile-background
+body-class: mobile
+order: 1
+published: true
+---
+
+# PyTorch Mobile
+
+Running ML on edge devices is growing in importance as applications continue to demand lower latency. It is also a foundational element for privacy-preserving techniques such as federated learning. As of PyTorch 1.3, PyTorch supports an end-to-end workflow from Python to deployment on iOS and Android. 
+
+This is an early, experimental release that we will be building on in several areas over the coming months:
+
+- Extends the PyTorch API to cover common preprocessing and integration tasks needed for incorporating ML in mobile applications
```

Comment:
extend (without 's')

And I'd maybe formulate it as "Provide APIs to cover common..." as it won't be part of PT core (more like torchvision for mobile)

Reply:
Sounds good. 

Let's go with:

- Provides APIs that cover common preprocessing...

Reply:
Updated